model-optimizer/extensions/ops/select_test.py

```diff
@@ -151,29 +151,33 @@ def test_select_infer_condition_shapes_broadcast(self, else_data_shape, than_dat
         self.assertTrue(flag, resp)
 
     @generate(*[
-        ([5, 6], [5, 6], [5, 6], np.ones(np.array([5, 6]), dtype=np.float),
-         np.zeros([5, 6], dtype=np.float), np.ones([5, 6], dtype=np.float),
-         np.ones([5, 6], dtype=np.float)),
-        ([15, 3, 5], [15, 1, 5], [15, 3, 5], np.ones([15, 3, 5], dtype=np.float),
-         np.zeros([15, 3, 5], dtype=np.float), np.ones([15, 3, 5], dtype=np.float),
-         np.ones([15, 3, 5], dtype=np.float)),
-        ([15, 3, 5], [15, 1, 5], [15, 3, 5], np.ones([15, 3, 5], dtype=np.float),
-         None, np.ones([15, 3, 5], dtype=np.float), np.ones([15, 3, 5], dtype=np.float)),
-        ([15, 3, 5], [15, 1, 5], [15, 3, 5], np.ones([15, 3, 5], dtype=np.float),
-         np.ones([15, 3, 5], dtype=np.float), None, None),
-        ([15, 3, 5], [15, 1, 5], [15, 3, 5], np.zeros([15, 3, 5], dtype=np.float),
-         None, np.ones([15, 3, 5], dtype=np.float), None),
-        ([15, 3, 5], [15, 1, 5], [15, 3, 5], np.zeros([15, 3, 5], dtype=np.float),
-         np.ones([15, 3, 5], dtype=np.float), None, np.ones([15, 3, 5], dtype=np.float)),
-        ([15, 3, 5], [15, 1, 5], [15, 3, 5], np.array([True], np.bool),
-         np.zeros([15, 3, 5], dtype=np.float), np.ones([15, 3, 5], dtype=np.float),
-         np.ones([15, 3, 5], dtype=np.float)),
-        ([15, 3, 5], [15, 1, 5], [15, 3, 5], np.array([False], np.bool),
-         np.zeros([15, 3, 5], dtype=np.float), np.ones([15, 3, 5], dtype=np.float),
-         np.zeros([15, 3, 5], dtype=np.float)),
+        ([5, 6], [5, 6], [5, 6], lambda x: np.ones(x, dtype=np.float),
```

Comment:
Please, explain why you changed this and what is the root cause of the original issue.

Reply:
We need to call numpy functions (zeros and ones) in a test function. Not in generator decorator.

Reply:
Please, add a comment in the test so we know why this workaround was implemented. It will be great to see some link describing the issue.